internal/dag/httpproxy_processor.go

```diff
@@ -100,6 +100,9 @@ type HTTPProxyProcessor struct {
 
 	// MaxRequestsPerConnection defines the maximum number of requests per connection to the upstream before it is closed.
 	MaxRequestsPerConnection *uint32
+
+	// PerConnectionBufferLimitBytes defines the soft limit on size of the listener’s new connection read and write buffers.
+	PerConnectionBufferLimitBytes *uint32
```

Comment:
I think we need to add this config to the Ingress and Gateway API processors as well to ensure this setting is applied to all generated Clusters

Reply:
Nice catch. Just updated the PR to include both the processors.